api/base/serializers.py

```diff
@@ -541,11 +545,13 @@ def resolve(self, resource):
         """
         Resolves the view for target node or target comment when embedding.
         """
-        view_info = self.view_map.get(resource.target._name, None)
+        view_info = self.view_map.get(resource.target.referent._name, None)
```

Comment:
A comment's target should always be a `Guid` object -- it's the `Guid.referent` that would refer to the comment, node or file